homeassistant/components/mazda/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Mazda Connected Services",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/mazda",
-  "requirements": ["pymazda==0.3.0"],
+  "requirements": ["pymazda==0.3.1"],
```

Comment:
I know the change is very small, but if it is not directly required for the binary sensors I would argue it could (should?) be a separate PR.

Reply:
Sure, I'm fine with making it a separate PR. Reverted this change.